web/download_manager.js

```diff
@@ -86,20 +86,20 @@ if (typeof PDFJSDev === 'undefined' || PDFJSDev.test('GENERIC || CHROME')) {
     },
 
     download: function DownloadManager_download(blob, url, filename) {
-      if (!URL) {
```

Comment:
Sounds good to me! I amended the commit message to be clearer about why I moved the `msSaveBlob` block.